src/deadlock-detection/lockorder.h

```diff
@@ -27,9 +28,13 @@ class CLockOrderTracker
     /// @var std::mutex lot_mutex
     /// mutex that is required to be locked before any method accesses any of this classes data members
     std::mutex lot_mutex;
+    /// @var std::map<void*, std::pair<std::string, boolean> >
+    // map for attempting to track the name of the mutex based on its reference. might not always be accurate, boolean
+    // will denote this
+    / std::map<void *, std::pair<std::string, bool> > mapMutexToName;
```

Comment:
`/` ?

Reply:
extra /. it seems the linter messed up the comments by removing / from previous lines and adding it to this one